src/components/Offers/New/CreateOfferForm.js

```diff
@@ -5,6 +5,15 @@ import { newOffer } from "../../../services/offerService";
 import useOfferForm from "../../../hooks/useOfferForm";
 import OfferForm from "../Form/form-components/OfferForm";
 import CreateOfferSchema from "./CreateOfferSchema";
+import { MailRegex } from "../../../utils/offer/OfferUtils";
+
+const parseApplyURL = (applyURL) => {
+    if (!applyURL)
+        return null;
+    if (MailRegex.test(applyURL) && /^(?!mailto:)/.test(applyURL))
+        return `mailto:${applyURL}`;
+    return applyURL;
+};
```

Comment:
this is duplicated, try to move it to somewhere else, maybe OfferUtils?

Reply:
Guess it makes sense.